docs/pipelines.md

```diff
@@ -900,6 +900,39 @@ This kind of variable can have any one of the values from the following table:
 
 For an end-to-end example, see [`status` in a `PipelineRun`](../examples/v1beta1/pipelineruns/pipelinerun-task-execution-status.yaml).
 
+### Using Aggregate Execution `Status` of All `Tasks`
+
+A `finally` task can utilize aggregate status of all `Tasks` using `params`:
+
+```yaml
+    finally:
+    - name: finaltask
+      params:
+        - name: aggreateTasksStatus
+          value: "$(tasks.status)"
+      taskSpec:
+        params:
+          - name: aggreateTasksStatus
+        steps:
+          - image: ubuntu
+            name: check-task-status
+            script: |
+              if [ $(params.aggreateTasksStatus) == "Failed" ]
+              then
+                echo "Looks like one or more tasks returned failure, continue processing the failure"
+              fi
+```
+
+This kind of variable can have any one of the values from the following table:
+
+| Status | Description |
+| ------- | -----------|
+| `Succeeded` | all `tasks` have succeeded |
+| `Failed` | one ore more `tasks` failed |
+| `Completed` | all `tasks` completed successfully including one or more skipped tasks |
+| `None` | no aggregate execution status available (i.e. none of the above), one or more `tasks` could be pending/running/cancelled/timedout |
+
+
```

Comment:
it'd be great to add a link to a working example demonstrating this

Reply:
yup, done 👍